sharedscriptfinal.sh

```diff
@@ -0,0 +1,20 @@
+#Anna's part
```

Comment:
1.	1
2.	The highest wage you report isn’t correct but your lowest wage and # females in top 10 are both correct. This is because of the sort call you have. You need to add the –n flag to it so that sort knows it’s sorting numerically
a.	0.9
3.	1
1.	1
2.	The highest wage you report isn’t correct but your lowest wage and # females in top 10 are both correct. This is because of the sort call you have. You need to add the –n flag to it so that sort knows it’s sorting numerically
a.	0.9
3.	1

2.9/3

Reply:
Looks like I pasted my comments twice, 1:3 are the ones to pay attention to. Sorry!